Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -742,6 +749,18 @@ void Wiimote::Update()
 			}
 		}
 
+		const ControlState sideways_toggle_pressed = m_hotkeys->controls[0]->control_ref->State();
+		const ControlState upright_toggle_pressed = m_hotkeys->controls[1]->control_ref->State();
+
+		if (sideways_toggle_pressed > s_sideways_toggle_last)
+			m_options->settings[1]->value = !(m_options->settings[1]->value);
+
+		if (upright_toggle_pressed > s_upright_toggle_last)
+			m_options->settings[2]->value = !(m_options->settings[2]->value);
+		
```

Comment:
Yes, Yes it can.

I think there is an option somewhere which changes that behavior.